ddr/lib/ddr-blobgen/java/genBinaryBlob.cpp

```diff
@@ -37,13 +37,18 @@
 #include <cstring>
 #include <fcntl.h>
 #include <limits.h>
+#include <limits>
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <vector>
 
+#ifndef UINT32_MAX
+#define UINT32_MAX std::numeric_limits<uint32_t>::max()
```

Comment:
I think its better to fully replace every instance of `UINT32_MAX` to avoid duplicate definitions on platforms that do support it.